### ISSUE TITLE
Add `v2` `balances` endpoint without usd pricing

### ISSUE THREAD

**Issue by Uxio0** [safe-global/safe-transaction-service#1998]
- Calculating token prices was deprecated so now service always return `0`
- People is confused as they think that's an error in our service
- A new `v2` `balances` endpoint without usd pricing should be added and `v1` flagged as deprecated

**Comment by moisses89** [comment]
You mean this endpoint? https://safe-transaction-sepolia.safe.global/api/v1/safes/0x0A64A32c010aa70fe79Ac32acAeFF0B3a7EE76b3/balances/?trusted=false&exclude_spam=false

**Comment by Uxio0** [comment]
You are right, let me rephrase it

**Comment by Uxio0** [comment]
Done, I'm taking this issue

**Comment by JagoFigueroa** [comment]
I have checked on staging and can confirm that the endpoint is gone + no issues on the UI. I will aks liliya to keep this in mind for mobile in case it would affect there
